guides/common/modules/performing_initial_configuration_sat_server_manual_disconnected.adoc

```diff
@@ -1,7 +1,7 @@
 [[performing_initial_configuration_sat_server_manual_disconnected]]
 = Performing the Initial Configuration Manually
 
-This initial configuration procedure creates an organization, location, user name, and password. After the initial configuration, you can create additional organizations and locations if required. The initial configuration also installs MongoDB and PostgreSQL databases on the same server. Depending on your deployment, using external databases can benefit performance. For more information see, xref:using-external-databases[].
+This initial configuration procedure creates an organization, location, user name, and password. After the initial configuration, you can create additional organizations and locations if required. The initial configuration also installs MongoDB and PostgreSQL databases on the same server. Depending on your deployment, using external databases can benefit performance. For more information see, link:{BaseURL}installing_satellite_server_from_a_connected_network/performing_additional_configuration_on_satellite_server#using-external-databases_satellite[Using External Databases with {Project}].
```

Comment:
```suggestion
This initial configuration procedure creates an organization, location, user name, and password. After the initial configuration, you can create additional organizations and locations if required. The initial configuration also installs MongoDB and PostgreSQL databases on the same server. Depending on your deployment, using external databases can benefit performance. For more information see, {BaseURL}installing_satellite_server_from_a_connected_network/performing_additional_configuration_on_satellite_server#using-external-databases_satellite[Using External Databases with {Project}].
```
Why did you add `link:` here? This is not common throughout our docs.

Reply:
Doesn't make a difference in the rendered output but does add highlighting to help me see these {BaseURL} links in my text editor :)

Reply:
Understood, please keep it then.